### PR TITLE
Initial version of a Weston based development product

### PR DESCRIPTION
**Pull request by andr2000**
Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>

### CODE REVIEW

File: doc/conf/local.conf [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+# and this option determines where those files are placed.
+#
+# You can wipe out TMPDIR leaving this directory intact and the build would regenerate
+# from these files if no changes were made to the configuration. If changes were made
+# to the configuration, only shared state files where the state was still valid would
+# be used (done using checksums).
+#
+# The default is a sstate-cache directory under TOPDIR.
+#
+SSTATE_DIR ?= "/media/storage/sstate-cache"
```
Comment (aanisov) [inline doc/conf/local.conf]: Is it really needed absolute path here?

File: doc/conf/local.conf [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+#
+# During a first build the system will download many different source code tarballs
+# from various upstream projects. This can take a while, particularly if your network
+# connection is slow. These are all stored in DL_DIR. When wiping and rebuilding you
+# can preserve this directory to speed up this part of subsequent builds. This directory
+# is safe to share between multiple builds on the same machine too.
+#
+# The default is a downloads directory under TOPDIR which is the build directory.
+#
+DL_DIR ?= "/media/storage/downloads"
```
Comment (aanisov) [inline doc/conf/local.conf]: Is it really needed an absolute path here?

Comment (andr2000) [inline doc/conf/local.conf]: yes, this is the path used by default on Xen-troops build server,
so you have less burden setting up automatic builds
but I can change it back to ${TOPDIR}/downloads

File: doc/conf/local.conf (hunk 1 above)
Comment (andr2000) [inline doc/conf/local.conf]: same as above

File: doc/conf/local.conf (hunk 2 above)
Comment (aanisov) [inline doc/conf/local.conf]: I would prefer relative path in a local.conf.
IMHO on a build server this should be easily redefined via command line of a build script by its runner.

Comment (andr2000) [inline doc/conf/local.conf]: done

File: doc/conf/local.conf (hunk 1 above)
Comment (andr2000) [inline doc/conf/local.conf]: done

**Pull request merged**